static/katex.less

```diff
@@ -442,10 +443,10 @@
     }
 
     .delimsizing {
-        &.size1 { font-family: KaTeX_Size1; }
```

Comment:
Good point. I never tried without it. I just looked at the code and said "hey, each of these should get a line height and I think one of the problems is solved" :grin: